modules.d/06hpsa/parse-hpsa.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin.bash
+### Laurence Oberman loberman@redhat.com
+### parse-hpsa.sh
+### Parses the rd.hpsa=x tp get the host number
+### Using rdloaddriver=hpsa will enforce hpsa becoming scsi0
+
+for p in $(getargs rd.hpsa=); do
+(
+     echo "echo 1 > /sys/class/scsi_host/host$p/rescan" > /sbin/hpsa_scan.sh
+    _do_hpsa=1
```

Comment:
Hi Harald ,
Its used a a flag and once the script is run its unset. The mechanism was taken from similar udev scripts. Perhaps its not needed, and was simply my lack of knowledge when I forst attempted this.

+#/bin/sh /sbin/hpsa.sh
+unset _do_hpsa
+